language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeIT.java

```diff
@@ -85,11 +86,11 @@
         analyzeApp.analyzeSyntax(
             "President Obama was elected for the second term");
 
-    List<String> got = token.stream().map(e -> e.getPartOfSpeech().getTag())
+    List<Tag> got = token.stream().map(e -> e.getPartOfSpeech().getTag())
```

Comment:
Yea - Java 8.   Good to see, but we don't typically use this in samples.  Note - I'm Ok w/ this.